app/views/rules_of_origin/steps/_not_wholly_obtained.html.erb

```diff
@@ -8,15 +8,15 @@
   </h1>
 
   <p class="govuk-body-l">
-    <%= t '.lead_para', trade_country_name: 'UK' %>
+    <%= t '.lead_para', trade_country_name: form.object.exporting? ? 'UK' : form.object.trade_country_name %>
```

Comment:
Could we consolidate this into a method on the object (possible the base step)?

Reply:
That'll be easier to validate than a view spec

Reply:
Sure thing